tools/python/transit/gtfs/download_gtfs.py

```diff
@@ -45,13 +45,17 @@ def download_gtfs_sources_mobilitydb(path):
             f"HTTP error {http_err} downloading zip from {URL_MOBILITYDB_GTFS_SOURCE}")
 
 
-def get_gtfs_urls_mobilitydb(path):
+def get_gtfs_urls_mobilitydb(path, countries_list):
     """Extracts the feed urls from the downloaded csv file"""
+    download_from_all_countries = False
```

Comment:
It's more logical to download all by default because downloading only a country is an optional argument.